Fix dotenv error in production

- [Error](https://github.com/acmcsufoss/lc-dailies/actions/runs/6078787872/job/16490442808#step:4:117):
```
error: Uncaught (in promise) MissingEnvVarsError: The following variables were defined in the example file but are not present in the environment:
  PORT, WEBHOOK_TOKEN, DISCORD_APPLICATION_ID, DISCORD_TOKEN, DISCORD_PUBLIC_KEY, DISCORD_WEBHOOK_URL, DISCORD_CHANNEL_ID

Make sure to add them to your env file.

If you expect any of these variables to be empty, you can set the allowEmptyValues option to true.
    throw new MissingEnvVarsError(
          ^
    at assertSafe (https://deno.land/std@0.201.0/dotenv/mod.ts:417:11)
    at load (https://deno.land/std@0.201.0/dotenv/mod.ts:336:5)
    at eventLoopTick (ext:core/01_core.js:183:11)
    at async file:///home/runner/work/lc-dailies/lc-dailies/cf/dailies/env.ts:7:1
Error: Process completed with exit code 1.
```

--- a/cf/dailies/env.ts
+++ b/cf/dailies/env.ts
@@ -4,7 +4,7 @@ import {
   DENOFLARE_VERSION_TAG,
 } from "lc-dailies/lib/denoflare/mod.ts";
 
-await load({ export: true, allowEmptyValues: true });
+await load({ export: true, examplePath: null });
 
 const CF_ACCOUNT_ID = Deno.env.get("CF_ACCOUNT_ID")!;
 const CF_API_TOKEN = Deno.env.get("CF_API_TOKEN")!;

--- a/cf/dailies/env.ts
+++ b/cf/dailies/env.ts
@@ -4,7 +4,7 @@ import {
   DENOFLARE_VERSION_TAG,
 } from "lc-dailies/lib/denoflare/mod.ts";
 
-await load({ export: true });
+await load({ export: true, allowEmptyValues: true });
 
 const CF_ACCOUNT_ID = Deno.env.get("CF_ACCOUNT_ID")!;
 const CF_API_TOKEN = Deno.env.get("CF_API_TOKEN")!;

